### PR TITLE
20171005 issue 93 - create a tool to make exceptions.json entries

### PR DESCRIPTION
**Pull request by mozmark**

### CODE REVIEW

File: entryMaker/main.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/mozilla/OneCRL-Tools/oneCRL"
+	"io/ioutil"
+)
+
+func check(e error) {
```
Comment (jcjones) [inline entryMaker/main.go]: I've always wondered why Golang hasn't adopted something like Rust's `?` syntax.

File: entryMaker/main.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
+package main
```
Comment (jcjones) [inline entryMaker/main.go]: license

File: entryMaker/main.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+)
+
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
+func main() {
+	certPtr := flag.String("cert", "", "a certificate file")
```
Comment (jcjones) [inline entryMaker/main.go]: nit: "a DER or PEM-encoded certificate file" right?

File: entryMaker/main.go (hunk 2 above)
Comment (mozmark) [reply entryMaker/main.go]: Oops. Yes.

File: entryMaker/main.go (hunk 1 above)
Comment (mozmark) [reply entryMaker/main.go]: Right. The idiomatic if err := ...; ... thing is great and all but the scoping makes for very deep / spacey code (though I do like how it forces you to be explicit)

**Comment by mozmark** [comment]
@jcjones is this OK with these changes?

**Pull request merged**